src/commands/appdistribution-group-create.ts

```diff
@@ -0,0 +1,17 @@
+import { Command } from "../command";
+import * as utils from "../utils";
+import { requireAuth } from "../requireAuth";
+import { AppDistributionClient } from "../appdistribution/client";
+import { getProjectName } from "../appdistribution/options-parser-util";
+
+export const command = new Command("appdistribution:group:create <displayName> [alias]")
```

Comment:
Should alias be a named option for clarity?

Reply:
I had it as an option before, but then changed my mind and decided against it. Mainly because it's an argument in `group:delete` - and I didn't want to change that.